package.json

```diff
@@ -264,10 +264,10 @@
     "@loadable/webpack-plugin": "5.14.0",
     "@pmmmwh/react-refresh-webpack-plugin": "0.4.3",
     "@sentry/cli": "1.41.2",
-    "@storybook/addon-actions": "6.2.0-beta.11",
-    "@storybook/addon-essentials": "6.2.0-beta.11",
-    "@storybook/addon-links": "6.2.0-beta.11",
-    "@storybook/react": "6.2.0-beta.11",
+    "@storybook/addon-actions": "6.3.7",
+    "@storybook/addon-essentials": "6.3.7",
+    "@storybook/addon-links": "6.3.7",
+    "@storybook/react": "6.3.7",
```

Comment:
Unrelated but story reloading was being weird; just ran the upgrade and seems to have fixed.